modules/app-gateway-module-backend/main.tf

```diff
@@ -58,8 +58,7 @@ resource "azurerm_application_gateway" "ag" {
   frontend_ip_configuration {
     name                          = "appGwPrivateFrontendIp"
     subnet_id                     = data.azurerm_subnet.app_gw.id
-    private_ip_address            = var.private_ip_address
-    private_ip_address_allocation = "Static"
+    private_ip_address_allocation = "Dynamic"
```

Comment:
this is a bit worrying doing this in a zero downtime way? wouldn't it be safer to allocate the IP address in each agw block in the yaml

Reply:
This has been set back to Static.